src/components/DayOverview/DayOverview.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import supabase, { Schema } from "@/supabase";
+import DayOverviewButton from "@/components/DayOverview/DayOverviewButton";
+
+interface Props {
+    text: string;
+}
+
+export type ParcelOfSpecificDateLocation = {
+    collection_datetime: Schema["parcels"]["collection_datetime"];
+    clients: Pick<
+        Schema["clients"],
+        "flagged_for_attention" | "full_name" | "address_postcode" | "delivery_instructions"
+    > | null;
+};
+
+const getParcelsOfSpecificDateAndLocation = async (
+    date: Date,
+    collectionCentre: string
+): Promise<ParcelOfSpecificDateLocation[]> => {
+    const startDateString = date.toISOString();
+    const endDate = new Date(date);
+    endDate.setDate(date.getDate() + 1);
+    const endDateString = endDate.toISOString();
+
+    const { data, error } = await supabase
+        .from("parcels")
+        .select(
+            "collection_datetime, clients(flagged_for_attention, full_name, address_postcode, delivery_instructions)"
+        )
+        .gte("collection_datetime", startDateString)
+        .lt("collection_datetime", endDateString)
+        .eq("collection_centre", collectionCentre)
+        .order("collection_datetime");
+
+    if (error) {
+        throw new Error(`Database error: ${JSON.stringify(error)}`);
+    }
+
+    return data;
+};
+
+const DayOverview = async ({ text }: Props): Promise<React.ReactElement> => {
+    const date = new Date("2023-07-17");
```

Comment:
Why is this hardcoded? Will we have a function to handle this later on or will this be more like exporting today's documents.

Reply:
Currently I want to set a specific date and specific location.
It is probably a future ticket issue if the button is designed to be the login user current date and the location the user belongs to
It is possible that people might also want to download the list for e.g. tomorrow - so I want to make it more flexible first
In the future maybe this can be extended to a panel letting user select a date on the calendar and a location (if they have the privilege to see others location for example?)